packages/at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -94,10 +94,11 @@ class AtLookupImpl implements AtLookUp {
   Future<bool> delete(String key,
       {String? sharedWith, bool isPublic = false}) async {
     var builder = DeleteVerbBuilder()
```

Comment:
at_lookup changes you can move to separate PR.
at_lookup dependency on at_commons 4.x is a minor version release for at_lookup?